scripts/common.js

```diff
@@ -335,6 +335,7 @@ async function getConstantValues() {
  * @returns {Object} An parsed object with those values and keys
  */
 export const extractObjectFromArray = (data) => {
+  if (!data || !Array.isArray(data)) return {};
```

Comment:
Just `!Array.isArray(data)` should be enough

Reply:
done